CHANGELOG.md

```diff
@@ -8,10 +8,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
   - [595](https://github.com/thoth-pub/thoth/issues/595) - Remove infrequently used and unused work statuses (unspecified, no longer our product, out of stock indefinitely, out of print, inactive, unknown, remaindered, recalled). Require a publication date for active, withdrawn from sale, and superseded works in Thoth. Add a new `Superseded` work status to replace Out of Print for older editions of Works. Require a withdrawn from sale date for Superseded works.
   - [582](https://github.com/thoth-pub/thoth/issues/582) - Add Crossmark metadata in Crossref DOI deposit when a Crossmark policy is present in the publisher record. Add Crossmark update new_edition metadata when a book is replaced by a new edition, and withdrawal metadata when a book is withdrawn from sale.
-  - [574](https://github.com/thoth-pub/thoth/issues/574) - Add descriptions to all remaining items in schema
+  - [574](https://github.com/thoth-pub/thoth/issues/574), [626](https://github.com/thoth-pub/thoth/pull/626) - Add descriptions to all remaining items in schema
 
 ### Fixed
   - [548](https://github.com/thoth-pub/thoth/issues/548) - Prevent users from deleting contributors/institutions which are linked to works by other publishers
+  - [626](https://github.com/thoth-pub/thoth/pull/626) - Rename `WorkStatus` `WITHDRAWN_FROM_SALE` to `WITHDRAWN`
```

Comment:
You can just merge this with L:9 (by adding the link to this PR to the other line)

Reply:
I meant, just add the PR number next to 595's (like in L:11) – there's no need to explain that you've renamed the work_status, you can just update the original message that mentions withdrawn from sale